test/functional/sqlsrv/TC21_Connection.phpt

```diff
@@ -35,8 +35,8 @@ function ConnectionTest()
     // Valid connection attempt => no errors are expected
     Trace("\nValid connection attempt (to $server) ....\n");
     $conn2 = Connect();
-    $errors = sqlsrv_errors(SQLSRV_ERR_ERRORS); 
-    if(count($errors) != 0)
+    $errors = sqlsrv_errors(SQLSRV_ERR_ERRORS);
+    if (!is_null($errors) && count($errors) != 0)
     {
```

Comment:
logically, should check if count($errors) > 0 instead of `count() != 0`? What do you think?